README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
```

Comment:
Please elaborate what this file is for. It is unclear what `jhhjkjhjkhkjs` represents.